composer.json

```
@@ -16,7 +16,7 @@
         "guzzlehttp/psr7": "^1.2"
     },
     "require-dev": {
-        "phpunit/phpunit": "~4.8.36",
+        "phpunit/phpunit": "^4.8|^5.0",
```

```suggestion
        "phpunit/phpunit": "^4.8.36|^5.6",
```

We need a PHPUnit version that already has the namespaced `TestCase` class alias.